requirements.txt

```diff
@@ -1,5 +1,3 @@
-openwisp-users @ https://github.com/openwisp/openwisp-users/tarball/master
+openwisp-users @ https://github.com/openwisp/openwisp-users/tarball/issues/348-django-4.2
 openwisp-utils[rest] @ https://github.com/openwisp/openwisp-utils/tarball/master
-django>=3.0.0,<4.1.0
```

Comment:
Removed the Django dependency to make it consistent with other OpenWISP modules.